### PR TITLE
update download_bin script

### PR DESCRIPTION
**Pull request by bxq2011hust**

### CODE REVIEW

File: tools/ci/download_bin.sh [hunk 1]
```diff
@@ -54,7 +57,12 @@ download_artifact()
     if [ ! -z "${download_gm}" -a -z "${is_gm}" ] || [ -z "${download_gm}" -a ! -z "${is_gm}" ]
```
Comment (cyjseagull) [inline tools/ci/download_bin.sh]: What's the function of `is_gm`, can it be deleted?

File: tools/ci/download_tar.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+{
+    local latest_version=$(curl -s https://api.github.com/repos/FISCO-BCOS/console/releases/latest | grep "tag_name" | sort -u | tail -n 1 | cut -d \" -f 4 | sed "s/^[vV]//")
+    mkdir -p ${output_dir}/console/releases/download/
+    local local_latest=$(ls -1 ${output_dir}/console/releases/download/ | sort -u | tail -n 1)
+    echo "latest console is ${latest_version}, local is ${local_latest}"
+    local dist_dir=${output_dir}/console/releases/download/v${latest_version}
+    if [ "v${latest_version}" != "${local_latest}" ];then
+        echo "download console ${latest_version} ..."
+        mkdir -p ${dist_dir}
+        local console_timeout=$(( timeout * 2 ))
```
Comment (cyjseagull) [inline tools/ci/download_tar.sh]: I think the `console_timeout` is a little small, you can test this value when deploy the shell script.

Comment (bxq2011hust) [reply tools/ci/download_tar.sh]: 10 mins are large, but you are right, we can test.

File: tools/ci/download_bin.sh (hunk 1 above)
Comment (bxq2011hust) [reply tools/ci/download_bin.sh]: no, it's a variable.

**Pull request merged**